geometry_msgs/Point corner11
geometry_msgs/Point corner12
geometry_msgs/Point corner21
geometry_msgs/Point corner22
geometry_msgs/Point high
geometry_msgs/Pose[] collision_object_pos
---
bool success_place